r/rsparkling/R/package.R

```diff
@@ -1,13 +1,24 @@
-
 #' @import sparklyr h2o utils
 NULL
 
 # define required spark packages
 spark_dependencies <- function(spark_version, scala_version, ...) {
+  sw_version = getOption("rsparkling.sparklingwater.version", default = "1.6.7")
```

Comment:
[this](https://0xdata.atlassian.net/browse/SW-209?focusedCommentId=37711&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-37711) chunk would avoid hardcoding sparkling version but adjust it to matching spark version. We still need to hardcode mapping of spark-sparkling versions.